### PR TITLE
Add PyCondor

### PR DESCRIPTION
**Pull request by jrbourbeau**
Adds PyCondor package to `conda-forge`

### CODE REVIEW

File: recipes/pycondor/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  home: https://github.com/jrbourbeau/pycondor
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Python utility for HTCondor'
+  description: |
+    PyCondor (Python HTCondor) is a tool to help build and submit workflows to
+    HTCondor in a straight-forward manner with minimal hassle.
+  doc_url: https://jrbourbeau.github.io/pycondor/
+  dev_url: https://jrbourbeau.github.io/pycondor/
```
Comment (synapticarbors) [inline recipes/pycondor/meta.yaml]: `dev_url` should be the repository: `dev_url: https://github.com/jrbourbeau/pycondor`

File: recipes/pycondor/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  requires:
+    - pytest
+  imports:
+    - pycondor
+    - pycondor.job
+    - pycondor.dagman
+    - pycondor.utils
+    - pycondor.basenode
+    - pycondor.cli
+    - click
```
Comment (synapticarbors) [inline recipes/pycondor/meta.yaml]: You don't need to import a dependency.

File: recipes/pycondor/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+requirements:
+  build:
+    - python >=2.7,<3.7
+    - pip
+  run:
+    - python >=2.7,<3.7
+    - click
+
+test:
+  requires:
```
Comment (synapticarbors) [inline recipes/pycondor/meta.yaml]: If you're not running tests, I'm guessing you don't need `pytest`

File: recipes/pycondor/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```
Comment (synapticarbors) [inline recipes/pycondor/meta.yaml]: It looks like there is a python entry point. Please add it:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

and a corresponding test:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

**Comment by jrbourbeau** [comment]
Thanks @synapticarbors!

**Pull request merged**